mitodl/ocw-hugo-themes
# feat: reorder resources in download course page

#### Pre-Flight checklist


#### What are the relevant tickets?
closes #958 

#### What's this PR do?
This PR orders resources by lecture number first, and then the remaining resources lexicographically.

#### How should this be manually tested?
Go to any menu with Download page, and see if lectures are on top and in ascending order. Same for other resources at bottom.

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
(Optional)
